app/controllers/posts_controller.rb

```diff
@@ -7,4 +7,26 @@ def show
     @user = User.find(params[:user_id])
     @post = Post.find(params[:id])
   end
+
+  def new
+    @post = Post.new
+
+    render :new, locals: { post: @post }
+  end
+
+  def create
+    post = current_user.posts.new(params.require(:post).permit(:title, :text))
```

Comment:
-[ ] Kindly use strong parameters here while creating posts for more security. 
```
 private 
    def post_params
      params.require(:post).permit(:title, :text)
    end
```
& use `post_params` in line 18.

Reply:
Strong parameter is now added to post creation! Thanks @Deepakdanger for the recommendation!